.github/workflows/build_deploy.yml

```diff
@@ -151,6 +155,16 @@ jobs:
           name: artifact
           path: dist
 
+      - uses: actions/checkout@v2
+        if: github.event.action == 'workflow_dispatch'
+        # Include all history and tags
+        with:
+          fetch-depth: 0
+      - name: Validate deploy version
+        if: github.event.action == 'workflow_dispatch'
+        run: |
+          ./scripts/validate-version "${{ github.event.inputs.expectedVersion }}"
+
       - uses: pypa/gh-action-pypi-publish@master
```

Comment:
Aside from your PR but i notice that we're pinned to master here which might leave us vulnerable to zero days if the github action is compromised